docs/rules/no-duplicate-case.md

```diff
@@ -0,0 +1,62 @@
+# Rule to disallow a duplicate case label (no-duplicate-case)
+
+A switch statements with duplicate case labels is normally an indication of a programmer error.
```

Comment:
We can't accept documentation with this little in the introduction. Show some examples, explain why it's hard to notice these on your own.